### PR TITLE
Fixed link in submission/comment deletion by mod PM & tiny Whoaverse -> Voat

### PR DESCRIPTION
**Pull request by MagicalBlob**

### CODE REVIEW

File: Whoaverse/Whoaverse/Controllers/CommentController.cs [hunk 1]
```diff
@@ -344,11 +344,11 @@ public ActionResult SubmitComment()
                 {
                     // notify comment author that his comment has been deleted by a moderator
                     MesssagingUtility.SendPrivateMessage(
-                        "Whoaverse",
+                        "Voat",
                         commentToDelete.Name,
                         "Your comment has been deleted by a moderator",
                         "Your [comment](/v/" + commentSubverse + "/comments/" + commentToDelete.MessageId + "/" + commentToDelete.Id + ") has been deleted by: " +
-                        "[" + User.Identity.Name + "](/u/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
+                        "[" + User.Identity.Name + "](/user/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
```
Comment (voat) [inline Whoaverse/Whoaverse/Controllers/CommentController.cs]: Both /u/ and /user/ are supported.

Comment (DanielFlamino) [inline Whoaverse/Whoaverse/Controllers/CommentController.cs]: I know both are supported, it's just that /user/ isn't affected by this
issue: https://github.com/voat/voat/issues/379

Comment (voat) [inline Whoaverse/Whoaverse/Controllers/CommentController.cs]: Okay, but the issue still remains. I think @PuttItOut could help look into this or maybe you could too, I think this is the relevant area: https://github.com/voat/voat/blob/master/Whoaverse/Whoaverse/Utils/Components/ContentFilters.cs#L51

Comment (DanielFlamino) [inline Whoaverse/Whoaverse/Controllers/CommentController.cs]: Yes, the issue I linked still remains (it occurs in any place that accepts both markdown and the user mentions), the one I "fixed" is pretty much just a side effect of that one.

**Comment by PuttItOut** [comment]
Why don't you just insert raw /u/username instead of the markdown? Won't the processor just match /u/username and convert to a link?

**Comment by DanielFlamino** [comment]
D'oh! Didn't even think about that. Fixed. Oh, @voat, can into merge now? :)

#379 still exists, though (but this wouldn't fix it so... yea).

**Comment by DanielFlamino** [comment]
[test comment pls ignore](www.google.com)

**Comment by DanielFlamino** [comment]
Again test comment, pls ignore

[Visit GitHub!](www.github.com)

**Pull request merged**